File: BaekJoon/BasicMath2/2581.py
import math


M = int(input())
N = int(input())

def isPrime(n):
    if n == 1 :
        return False
    
    for i in range(2, math.floor(math.sqrt(n))+1):
        if n%i == 0:
            return False
    return True


result = []

for i in range(M,N+1):
    if isPrime(i) == True:
        result.append(i)

if len(result) == 0:
    print(-1)
else:
    print(sum(result))
    print(result[0])

File: BaekJoonSwift/BaekJoonSwift/Basic Math2/2581.swift
import Foundation

func isPrime(num : Int) -> Bool {
    var result = true
    if num == 1 {
        return false
    }
    
    if num == 2 || num == 3 {
        return true
    }
    
    for i in 2...Int(sqrt(Double(num))) {
        if num % i == 0 {
            result = false
            break
        }
    }
    
    return result
}

let a = Int(readLine()!)!
let b = Int(readLine()!)!
var minValues = -1
var sum = 0

for i in a...b {
    if isPrime(num: i) {
        sum += i
        if minValues == -1 {
            minValues = i
        }
    }
}

if minValues != -1 {
    print(sum)
    print(minValues)
} else {
    print(-1)
}
